pkg/utils/installRbac.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+var (
+	err    error
+	out    bytes.Buffer
+	stderr bytes.Buffer
+)
+
+// InstallRbac function is for the Rbac Creation for indivisual experiments. we need to pass the address
+// (From the chaos chart) of the Rbac file and the name of experiment and it will create the Rbac
+// for the file.
+func InstallRbac(rbacPath string, rbacNamespace string, experimentName string, client *kubernetes.Clientset) error {
+
+	//Installing RBAC for the experiment
+	//Fetching RBAC file
+	err = DownloadFile(experimentName+"-sa.yaml", rbacPath)
+	if err != nil {
+		return errors.Wrapf(err, "Fail to fetch the experiment file, due to %v", err)
+	}
+	//Modify Namespace field of the RBAC
+	err = EditFile(experimentName+"-sa.yaml", "namespace: default", "namespace: "+rbacNamespace)
+	if err != nil {
+		return errors.Wrapf(err, "Fail to Modify experiment file, due to %v", err)
+	}
+	cmd := exec.Command("kubectl", "apply", "-f", experimentName+"-sa.yaml")
```

Comment:
Use k8s client to apply the yaml

Reply:
I will be creating an issue for this ... will cover in a separate PR